templates/assopy/invoices/_additional_text_for_2021.html

```diff
@@ -0,0 +1,9 @@
+<p style='margin-top: 2em;'>
+The EuroPython 2020 conference is held in Ireland. Accordingly, we have to charge Irish VAT on the total amount.
```

Comment:
is it still the case with the VAT?

Reply:
and with the location of the conference.

Reply:
Yes, for EP2021 :-)

Reply:
But good point. I need to fix the text line as well.